app/src/ui/tutorial-panel/tutorial-panel.tsx

```diff
@@ -170,17 +206,42 @@ class TutorialListItem extends React.PureComponent<{
 }> {
   public render() {
     return (
-      <li key={this.props.sectionId}>
+      <li key={this.props.sectionId} onClick={this.onClick}>
         <details
           open={this.props.sectionId === this.props.currentlyOpenSectionId}
           onClick={this.onClick}
         >
-          <summary>{this.props.summaryText}</summary>
-          {this.props.children}
+          {this.renderSummary()}
+          <div className="contents">{this.props.children}</div>
         </details>
       </li>
     )
   }
+
+  private renderSummary = () => (
+    <summary>
+      {this.renderTutorialStepIcon()}
+      <span className="summary-text">{this.props.summaryText}</span>
+      <Octicon className="chevron-icon" symbol={OcticonSymbol.chevronDown} />
+    </summary>
+  )
+
+  private renderTutorialStepIcon = () => {
```

Comment:
No need for this to be an anonymous instance function right?
```suggestion
  private renderTutorialStepIcon() {
```

Reply:
ah right, we're not passing it anywhere